### ISSUE TITLE
Detect and exclude random selectors

### ISSUE THREAD

**Issue by fregante** [fczbkk/css-selector-generator#213]
With more and more websites using CSS modules, `[css-selector-generator](https://github.com/fczbkk/css-selector-generator)` often generates fickle selectors that are only valid until the next deployment.

Would it be possible to automatically detect and exclude these selectors, optionally?

<img width="140" alt="Screen Shot 15" src="https://user-images.githubusercontent.com/1402241/154790124-a7b44d80-863c-4125-90bb-0e578661c583.png"> <img width="256" alt="Screen Shot 14" src="https://user-images.githubusercontent.com/1402241/154790125-ba959bb1-8529-4b14-a1e5-7385f2d3bb37.png"> <img width="270" alt="Screen Shot 16" src="https://user-images.githubusercontent.com/1402241/154790119-aa4320f5-6586-4b22-9a27-e7ae20c962cf.png">

**Comment by fczbkk** [comment]
@fregante You can use blacklist to filter these selectors:
https://github.com/fczbkk/css-selector-generator#blacklist

I already filter Angular attributes when generating attribute selectors:
https://github.com/fczbkk/css-selector-generator/blob/f9eece2856fca7125d34ffc09b85a2ca78758173/src/selector-attribute.ts#L12-L13

I'm not sure about hard-coding patterns for classnames like these into the library itself. Seems to me that it could lead to unforeseen false-positives. Maybe I could put together some examples of blacklists for specific libraries and list them in the Readme, to make it easy for users of this library to use them at their own risk. What do you think?

**Comment by biels** [comment]
Yes, add a callback to determine whether a class / id should be used in a selector. The user can use the method that suits best for him. For example detecting the entropy of the string, or using a whitelist, a regex, or any other method
